doc/OnlineDocs/getting_started/index.rst

```diff
@@ -1,9 +1,58 @@
 Getting Started
 ===============
 
-Installation, pyomo command, a simple example, etc
+Installation
+------------
 
-.. doctest::
+Pyomo currently supports the following versions of Python:
+
+* CPython: 2.7, 3.5, 3.6
+
```

Comment:
@whart222 you're missing 3.4 in this list

Reply:
Done.